screens/myListsScreen.tsx

```diff
@@ -63,6 +65,11 @@ class myListsScreen extends React.Component<Props, State> {
   }
   componentDidMount() {
     this.fetchLists();
+    setTimeout(() => {
+      this.setState({loading: false});
+    }, 250)
```

Comment:
This should be removed. The screen should only be in the loading state during the API request.